toolflow/vivado/arch/axi4mm/axi4mm.tcl

```diff
@@ -384,8 +378,8 @@ namespace eval arch {
       [get_bd_pins -of_objects [get_bd_cells] -filter "TYPE == rst && NAME =~ *interconnect_aresetn && DIR == I"]
     connect_bd_net [tapasco::subsystem::get_port "design" "rst" "peripheral" "resetn"] \
       [get_bd_pins -of_objects [get_bd_cells -of_objects [current_bd_instance .]] -filter "TYPE == rst && NAME =~ *peripheral_aresetn && DIR == I"] \
-      [get_bd_pins -filter { TYPE == rst && DIR == I && CONFIG.POLARITY != ACTIVE_HIGH } -of_objects [get_bd_cells -filter {NAME =~ "target_ip*"}]]
-    set active_high_resets [get_bd_pins -of_objects [get_bd_cells] -filter "TYPE == rst && DIR == I && CONFIG.POLARITY == ACTIVE_HIGH"]
+      [get_bd_pins -filter { TYPE == rst && DIR == I && CONFIG.POLARITY != ACTIVE_HIGH } -of_objects [get_bd_cells -hier -filter {PATH =~ "*target_ip*"}]]
+    set active_high_resets [get_bd_pins -of_objects [get_bd_cells -hier] -filter "TYPE == rst && DIR == I && CONFIG.POLARITY == ACTIVE_HIGH"]
```

Comment:
It seems there are inconsistencies how the relevant bd_cells are selected. When looking at the lines 380, 381 and 382 why are there three different calls to get_bd_cells?

Reply:
Same here, first all the top level modules in `/arch`, then all PEs within the wrapper. For ACTIVE_HIGH, we don't have to exclude the interconnect trees.

Reply:
Make sense